readme.md

```diff
@@ -76,6 +76,19 @@ Specific tag to generate changelog against.
 
 None.
 
+## Releasing previous tags
```

Comment:
This should be _"Release __notes__ for previous tags/releases"_ or something along those lines. Thoughts?

Reply:
```suggestion
## "Release notes for previous tags/releases
```